backend/migrations/0003_create_users_table.up.sql

```diff
@@ -0,0 +1,11 @@
+-- create_users_table.up
+-- Prefer TEXT > VARCHAR: https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_defaultj
+CREATE TABLE IF NOT EXISTS users (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS user(
```
https://www.sqlshack.com/learn-sql-naming-conventions/

Reply:
table names should be singular

Reply:
Changed to user_profile because user is a reserved pgsql [keyword](https://www.postgresql.org/docs/9.3/sql-keywords-appendix.html)